baseclasses/utils.py

```diff
@@ -73,5 +74,4 @@ def __init__(self, message):
                 msg += word + " "
                 i += len(word) + 1
         msg += " " * (78 - i) + "|\n" + "+" + "-" * 78 + "+" + "\n"
-        print(msg)
-        super().__init__()
+        super().__init__(msg)
```

Comment:
Does this print the message at all? I tried it on ADflow and it doesn't work.

Reply:
Huh really? It works fine with me. Try the following MWE?
```
from baseclasses.utils import Error
raise Error('Failed!')
```

Reply:
That works. Maybe it has something to do with Fortran.

Reply:
pyHyp and IDWarp both print the error messages, so it might just a problem with ADflow.

Reply:
Okay, it seems like the problem was on my end with running a `unittest` script with `python`. This works for me on a regular ADflow run script or for a `unittest` script with `testflo`.